package.json

```diff
@@ -22,5 +22,12 @@
   "bugs": {
     "url": "https://github.com/mok-labs/locales-check/issues"
   },
-  "homepage": "https://github.com/mok-labs/locales-check#readme"
+  "homepage": "https://github.com/mok-labs/locales-check#readme",
+  "dependencies": {},
+  "devDependencies": {
+    "@types/jest": "^29.5.12",
+    "cac": "^6.7.14",
```

Comment:
cac looks awesome ❤️

Reply:
It worked link a dev dependency when installing it in other project withput cac. This package should only be used in development or ci I thought that it was the right place to put it? But maybe I am missunderstanding the meaning ofd dev dependencies

Reply:
Wait you were right. I was propably trying it in an older version. As devdependency here I get an error in the other repo without cac. I will change it!

Reply:
✅  solved

Reply:
Thanks for the explanation!